ecpp

```diff
@@ -1,35 +1,75 @@
 #!/usr/bin/python
 
+# Author: Hubert Kario, Stefan Dordevic
+# Released under Gnu GPL v2.1, see LICENSE file for details
+
 from __future__ import print_function
 import ConfigParser
 import sys
 import StringIO
 from verify import *
+from convert import *
 import getopt
+import os
+import re
 
 # xrange python3 compatibility
 try:
     xrange
 except NameError:
     xrange = range
 
+# input files directory for primo tool
+PRIMO_IN = 'in/'
+CERTS_DIR = 'certificates/'
+
 # dict with params
-params = {}
+PARAMS = {}
 def par_set():
-    params.update({'s':None, 'w':None, 'j':None, 't':None, 'a':None, 'b':None, 'q':None, 'r':None})
+    """Paramiter set/reset function"""
+    PARAMS.update({'s':None, 'w':None, 'j':None, 't':None,
+                   'a':None, 'b':None, 'q':None, 'r':None})
 
 # division function
 def is_prime_64(n):
+    """Check if number is a prime through division function"""
     if n % 2 == 0:
         raise ValueError("not prime")
     for i in xrange(3, int(n**0.5)+1, 2):
         if n % i == 0:
             raise ValueError("not prime: " + str(n))
 
-def main():
-    with open(sys.argv[1], 'r+') as f:
+def candidate_finder(path):
+    """Return the int number from certificates Candidate section.
+       If file is malformed, file is not primality certificate, or version
+       of primality certificate is unsupported 'None' will be returned"""
+    with open(path) as f:
         config_buff = f.read()
         f.close()
```

Comment:
when `with open()` is used, then `f.close()` is not needed

also, if the contents of the file are in an array, then it's not necessary to put all method code inside the `with` section